latch_cli/snakemake/serialize.py

```diff
@@ -423,6 +425,8 @@ def generate_jit_register_code(
         from latch.types.directory import LatchDir
         from latch.types.file import LatchFile
 
+        from latch_metadata import *
```

Comment:
maybe load this by path? not sure

most certainly don't want `import *` if we can at all avoid it